x-pack/plugins/security_solution/public/common/components/exceptions/add_exception_modal/index.tsx

```diff
@@ -128,6 +133,18 @@ export const AddExceptionModal = memo(function AddExceptionModal({
     memoSignalIndexName
   );
 
+  const { loading: isLoadingAlertData, data: alertData } = useFetchAlertData(
+    ecsData?._id,
+    signalIndexName
+  );
+
+  const alert = useMemo(() => {
+    if (isLoadingAlertData === false && ecsData != null) {
+      const { _id, _index, _source } = alertData?.hits.hits[0] || {};
```

Comment:
I think this would throw an error if `alertData.hits.hits[0]` is `null` or `undefined`